easybuild/easyconfigs/z/zlib/zlib-1.2.8-goolf-1.7.20.eb

```diff
@@ -0,0 +1,21 @@
+easyblock = 'ConfigureMake'
+
+name = 'zlib'
+version = '1.2.8'
+
+homepage = 'http://www.zlib.net/'
+description = """zlib is designed to be a free, general-purpose, legally unencumbered -- that is,
+ not covered by any patents -- lossless data-compression library for use on virtually any
+ computer hardware and operating system."""
+toolchain = {'name': 'goolf', 'version': '1.7.20'} # tweaked by EasyBuild (was: {'name': 'foss', 'version': '2014b'})
```

Comment:
he may as well drop the whole file, since it is provided via:
https://github.com/hpcugent/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/z/zlib/zlib-1.2.8-goolf-1.7.20.eb
(I think sanest way with git is to simply copy-paste that strain over here)